gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeStatisticsPanel.java

```diff
@@ -280,6 +316,11 @@ private double calculateVariance(final List<Double> allGrades) {
 	private double calculateStandardDeviation(final List<Double> allGrades) {
 		return Math.sqrt(calculateVariance(allGrades));
 	}
+
+	private boolean isExtraCredit(Double grade, Assignment assignment) {
+		return (GbGradingType.PERCENTAGE.equals(gradingType) && grade > 100) ||
```

Comment:
does this work if the points calculated down from the percentage is greater than 100? ie relative weight of 200, give someone 75% = 150 points.